[Web Analytics] Changed data retention to 6 months FAQ (#4142)
cloudflare/cloudflare-docs: content/analytics/faq/web-analytics/index.md

# Web Analytics FAQ

> Below you will find answers to our most commonly asked questions. If you cannot find the answer you are looking for, heading over to our [community page](https://community.cloudflare.com/) is always a good idea.

Reviewer: ```suggestion
Below you will find answers to our most commonly asked questions. If you cannot find the answer you are looking for, refer to the [community page](https://community.cloudflare.com/) to explore more resources.
```

Reviewer: "Heading over" borders jargon-y because it won't translate helpfully from English to other languages.
